crates/bevy_math/src/bounding/bounded3d/primitive_impls.rs

```diff
@@ -461,7 +462,10 @@ mod tests {
 
         let bounding_sphere = polyline.bounding_sphere(isometry);
         assert_eq!(bounding_sphere.center, translation.into());
-        assert_eq!(bounding_sphere.radius(), 1.0_f32.hypot(1.0).hypot(1.0));
+        assert_eq!(
+            bounding_sphere.radius(),
+            ops::hypot(ops::hypot(1.0, 1.0), 1.0)
```

Comment:
These chained hypots are pretty cursed, but I guess it's fine inside a test.

Reply:
Yeah, I had the same thought...